tests/unittests/OperatorTest.cpp

```diff
@@ -3003,7 +3003,40 @@ TEST_P(OperatorTest, TopK1) {
   EE_.run(bindings_);
 
   auto V = bindings_.get(values->getPlaceholder())->getHandle();
-  auto I = bindings_.get(indices->getPlaceholder())->getHandle<sdim_t>();
+  auto I = bindings_.get(indices->getPlaceholder())->getHandle<int64_t>();
+
+  EXPECT_FLOAT_EQ(V.at({0, 0, 0}), 18);
+  EXPECT_EQ(I.at({0, 0, 0}), 1);
+  EXPECT_FLOAT_EQ(V.at({1, 0, 0}), 41);
+  EXPECT_EQ(I.at({1, 0, 0}), 3);
+  EXPECT_FLOAT_EQ(V.at({2, 0, 0}), 34);
+  EXPECT_EQ(I.at({2, 0, 0}), 2);
+}
+
+// Check the TopK operator for the special case of K=1.
+TEST_P(OperatorTest, TopK1int32) {
```

Comment:
nit: Appears to be nearly identical to `TopK1` -- can we templatize the original to consolidate them?